test/elife-loader.spec.js

```diff
@@ -0,0 +1,92 @@
+const chai = require('chai');
+const sinon = require('sinon');
+
+const expect = chai.expect;
+const spy = sinon.spy;
+
+// load in component(s) to be tested
+let loader = require('../assets/js/elife-loader');
+
+describe('The eLife Loader', function () {
+  "use strict";
+
+  let loader;
```

Comment:
Doesn't look to be used anywhere, delete.

Reply:
Completed